util/testutil/integration/sandbox.go

```diff
@@ -18,6 +22,8 @@ import (
 
 const buildkitdConfigFile = "buildkitd.toml"
 
+const maxSandboxTimeout = 5 * time.Minute
```

Comment:
5min might not be enough for this test `TestIntegration/TestBasicGhaCacheImportExport`: https://github.com/moby/buildkit/actions/runs/12189447998/job/34004770018?pr=5568#step:8:2218

Default timeout is 10m: https://github.com/moby/buildkit/blob/409f7b4657010c2226f1e8d43bd7edc45b5ade6b/cache/remotecache/gha/gha.go#L44

Reply:
Added extra time to GHA test